modelforge/tests/test_painn.py

```diff
@@ -1,11 +1,89 @@
+import os
+
+import pytest
+
 from modelforge.potential.pain import PaiNN
+from modelforge.potential.utils import CosineCutoff
 
 from .helper_functions import generate_methane_input
 
-from modelforge.potential.utils import CosineCutoff
+IN_GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") == "true"
 
 
 def test_PaiNN_init():
     """Test initialization of the PaiNN neural network potential."""
     painn = PaiNN(128, 6, 10, cutoff_fn=CosineCutoff(5.0))
     assert painn is not None, "PaiNN model should be initialized."
+
+
+@pytest.mark.skipif(
+    IN_GITHUB_ACTIONS,
+    reason="This test is not intended to be performed regularly.",
+)
+def test_schnetpack_PaiNN():
+    import os
+
+    import schnetpack as spk
+    import schnetpack.transform as trn
+    from schnetpack.datasets import QM9
+
+    from .schnetpack_pain_implementation import setup_painn
+
+    qm9tut = "./qm9tut"
+    if not os.path.exists("qm9tut"):
+        os.makedirs(qm9tut)
+
+    qm9data = QM9(
+        "./qm9.db",
+        batch_size=64,
+        num_train=1000,
+        num_val=1000,
+        transforms=[
+            trn.ASENeighborList(cutoff=5.0),
```

Comment:
we are using nm, I'm assuming this is angstroms which would be inconsistent.  We should make sure that we are commenting and annotating "This calls schnetpack which uses different units..., the unit system is...")